apps/meeting/serverless/src/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "chime-sdk-meeting-serverless-demos",
+  "version": "0.1.0",
+  "description": "Amazon Chime SDK Serverless Demos",
+  "dependencies": {
+    "aws-sdk": "^2.1034.0"
+  },
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/aws-samples/amazon-chime-sdk"
+  }
+}
```

Comment:
Why do we need this `package.json`? Does the one under `/meeting` folder not work? I think we should avoid this if possible.

Reply:
if no this package.json, the aws-sdk retrieved is of version 2.1001.0, which will have the error `AWS.ChimeSDKMeetings is not a constructor`.